templates/vue/src/components/Lightbox/media/VideoMedia/UrlVideoMedia.vue

```diff
@@ -126,6 +127,18 @@ export default {
         currentTime,
       })
     },
+    handleError() {
+      // Error code 4 - The associated resource or media provider object has been found to be unsuitable.
+      if (this.$refs.video.error.code === 4) {
+        if (
```

Comment:
Should we automatically try and fetch this node from the server (once only) and use its updated media URL before showing the refresh option?

Reply:
Done